doc/content/lorawan/end-device-activation/index.md

```diff
@@ -0,0 +1,403 @@
+---
+title: End Device Activation
+section: The Things Fundamentals
+weight: 60
+---
+
+Every end device must be registered with a network before sending and receiving messages. This procedure is known as **activation**. There are two activation methods available:
+
+* **Over-The-Air-Activation (OTAA)** - OTAA is the most secure activation method for end devices. Devices perform a join-procedure with the network, during which a dynamic Device Address is assigned and security keys are negotiated with the device.
+* **Activation By Personalization (ABP)** - Requires hardcoding the Device Address as well as the security keys in the device. ABP is less secure than and also has the downside that devices can not switch network providers without manually changing keys in the device. 
```

Comment:
Doesn't really require hardcoding the device, but the device is already hardcoded (if we are talking from a user perspective)